x/delegation/keeper/un_delegation_state.go

```diff
@@ -210,12 +196,14 @@ func (k Keeper) IncrementUndelegationHoldCount(ctx sdk.Context, recordKey []byte
 	return nil
 }
 
+// GetUndelegationHoldCount returns the hold count for the undelegation record key.
 func (k *Keeper) GetUndelegationHoldCount(ctx sdk.Context, recordKey []byte) uint64 {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.GetUndelegationOnHoldKey(recordKey))
 	return sdk.BigEndianToUint64(bz)
 }
 
```

Comment:
Not necessary because:
```go
// BigEndianToUint64 returns an uint64 from big endian encoded bytes. If encoding
// is empty, zero is returned.
func BigEndianToUint64(bz []byte) uint64 {
	if len(bz) == 0 {
		return 0
	}

	return binary.BigEndian.Uint64(bz)
}
```